openapi/index.yaml

```diff
@@ -66,3 +66,85 @@ definitions:
       - statusQueryGetUri
       - sendEventPostUri
       - terminatePostUri
+  ConsultazioneSogliaIndicatoreInput:
+    type: object
+    properties:
+      CodiceFiscale:
+        type: string
+        pattern: ([A-Za-z]{6}\\d{2}[A-Za-z]\\d{2}[A-Za-z][A-Za-z0-9]{3}[A-Za-z])|(\\d{11} {0,5})
+      CodiceSoglia:
+        type: string
+        pattern: ([A-Za-z]{4}[0-9]{2})
+      FornituraNucleo:
+        type: string
+        enum:
+          - SI
+          - "NO"
+      DataValidita:
+        type: string
+        pattern: ( \\d{4}-[01][0-9]-[0123][0-9])
+    required:
+      - CodiceFiscale
+      - CodiceSoglia
+      - FornituraNucleo
+  ConsultazioneSogliaIndicatoreResponse:
+    type: object
+    properties:
+      Componente:
+        type: array
+        items:
+          $ref: "#/definitions/NucleoType"
+        x-nullable: true
+      TipoIndicatore:
+        type: string
+        enum:
+          - ISEE Standard
+          - ISEE Minorenne ordinario
+          - ISEE Minorenne con componente aggiuntiva
+          - ISEE Minorenne con componente attratta
+          - ISEE Universitario con componente aggiuntiva
+          - ISEE Universitario con componente attratta
+          - ISEE Universitario ordinario
+          - ISEE Universitario con studente attratto
+          - ISEE Universitario con studente attratto e componente aggiuntiva
+          - ISEE Universitario con studente attratto e componente attratta
+          - ISEE Sociosanitario ridotto
+          - ISEE Dottorato ricerca ridotto
+          - ISEE Residenziale standard senza figli aggiuntivi
+          - ISEE Residenziale standard con figli aggiuntivi
+          - ISEE Residenziale ridotto senza figli aggiuntivi
+          - ISEE Residenziale ridotto con figli aggiuntivi
+      SottoSoglia:
+        type: string
+        enum:
+          - SI
+          - "NO"
+      ProtocolloDSU:
+        type: string
+        pattern: "INPS-ISEE-[0-9]{4}-[0-9]{8}[A-Z]-[0-9]{2}"
+      DataPresentazioneDSU:
+        type: string
+        pattern: \\d{4}-[01][0-9]-[0123][0-9]
+    required:
+      - TipoIndicatore
+      - SottoSoglia
+      - ProtocolloDSU
+      - DataPresentazioneDSU
+  NucleoType:
+    type: object
+    properties:
+      CodiceFiscale:
+        type: string
+        pattern: '([A-Z]{6}\\d{2}[A-Z]\\d{2}[A-Z][A-Z0-9]{3}[A-Z])|(\\d{11} {0,5})'
+      Cognome:
+        type: string
+        pattern: "[A-ZÀÈÉÌÒÙ' -]*"
```

Comment:
we don't need this kind of validation, t.string is enough

Reply:
fixed in 0c41302